jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialWriter.kt

```diff
@@ -56,6 +57,9 @@ class CreateOrUpdateCredentialProfilesAction @TestOnly constructor(
         localFileSystem.refreshFiles(listOf(virtualFile), false, false) {
             fileEditorManager.openTextEditor(OpenFileDescriptor(project, virtualFile), true)
                 ?: throw RuntimeException(message("credentials.could_not_open", file))
+
+            // TODO : remove message (and localized string) when credentials auto-refreshing is supported
+            notifyInfo("", message("credentials.notification.restart.ide"))
```

Comment:
Do we need to provide a title for this info?

Reply:
I don't feel that it is necessary
![image](https://user-images.githubusercontent.com/39839589/49391700-40cc3780-f6e1-11e8-8d92-888a147ac2f3.png)

Reply:
From this message, it's not clear to tell what the credentials are, although it could be inferred from the context. Maybe we should change the `credentials` to `AWS credentials` to be more clear?

Reply:
`AWS credential`